src/checkstatus/get_dbstatus_function.py

```diff
@@ -2,9 +2,14 @@
 import boto3
 import constants
 import utility as util
+import logging
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
is the setting of logging type explicitly needed or based on the logger methods can the logging type be inferred?

Reply:
@namitad I did more research and no it isn't needed. It is used to set the lowest severity that can be logged.

Reply:
@namitad on a second note, I actually need to set the level because without setting it the default lowest severity is WARNING which is higher than INFO. Hence, all INFO logs are ignored. See https://docs.python.org/2/howto/logging.html

Reply:
@zuliatowoade have you tested it out without setting the log level explicitly? to see if the logs are still published to cloudwatch?